### DISCUSSION TITLE
Welcome to OSXChatGpt Discussions!

### DISCUSSION THREAD

**Discussion by MustangYM** [MustangYM/OSXChatGPT#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WeezyShady** [comment]
希望能被看见~~！

不知道为什么，之前都用得好好的。中途更新过一次API密钥后，不停提示“Response Error code:429 res: You exceeded your current quota, please check your plan and billing details. ”其实除去更新过密钥之外，其余操作都没有做过，openAI里的流量也没过载。不知道有人遇到过吗。

**Comment by MustangYM** [comment]
看起来是你的账号token消耗尽了。


| |
mustangym
|
|
***@***.***
|


---- Replied Message ----
| From | ***@***.***> |
| Date | 12/24/2023 21:43 |
| To | ***@***.***> |
| Cc | ***@***.***> ,
***@***.***> |
| Subject | Re: [MustangYM/OSXChatGpt] Welcome to OSXChatGpt Discussions! (Discussion #2) |

希望能被看见~~！

不知道为什么，之前都用得好好的。中途更新过一次API密钥后，不停提示“Response Error code:429 res: You exceeded your current quota, please check your plan and billing details. ”其实除去更新过密钥之外，其余操作都没有做过，openAI里的流量也没过载。不知道有人遇到过吗。

**Comment by WeezyShady** [comment]
啊啊啊大神！！我不是很理解token的概念....那token被消耗尽了的话我能怎么处理嘛？

SixOne ***@***.***> 于2023年12月26日周二 01:49写道：
